packages/zosfiles/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe z/OS files SDK package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Fixed error where `Get.dataSet` and `Get.USSFile` could silently fail when downloading large data sets or files. [#2167](https://github.com/zowe/zowe-cli/pull/2167)
```

Comment:
Can we say what Get.dataSet and Get.USSFile are? Profiles? Classes? REST APIs? Cat breeds?